Update metrics widget image (#692)
elastic/observability-docs: docs/en/observability/observability-ui.asciidoc

[[view-system-metrics]]
== System metrics

> The *Metrics* chart provides you with a glance at your top hosts with the most significant
> resource footprint. The metrics help you evaluate the host efficiency and determine whether
> there is any impact of resource consumption for your end-users.

Reviewer: "provides you with a glance at" seems wordy to me. What about something like this?

```suggestion
The *Metrics* chart displays your top hosts with the most significant
resource footprints. These metrics help you evaluate host efficiency and determine if
resource consumption is impacting end-users.
```

Author: Great suggestion. Thanks!
